### ISSUE TITLE
ppl positions hinzufügen bei ehemaligen mitarbeitern / professoren

### ISSUE THREAD

**Issue by sonschul** [FISvirtUOS/VIVO#40]
MEMO an mich selbst: 

A spreadsheet of people’s positions in the organization will look like this:

![image](https://user-images.githubusercontent.com/40201793/51603236-1b99bf80-1f01-11e9-99a3-5951ea98bf30.png)

As one can notice in this spreadsheet we use the UID of the person and the org_ID of the organizational unit to connect the person with that unit and create the RDF statement containing information about that person’s position. 

![image](https://user-images.githubusercontent.com/40201793/51603226-15a3de80-1f01-11e9-8e0d-c368e9cdec59.png)

**Comment by dofeldsc** [comment]
Das betrifft die aktuellen Datenlieferung noch nicht oder?

**Comment by sonschul** [comment]
Nein, das ist in der aktuellen Lieferung zu den Personen noch nicht enthalten. 
Wir hatten überlegt die Rufe bzw. Wechsel z.B. von W1 zu W2 als Positions einzupflegen. Diese Tabelle gibt es aber nicht nicht.
